### PR TITLE
Add sentences for #341

### PR DESCRIPTION
**Pull request by mds3dstn71**
These changes will add sentences from:

- [Ceejaydeepee](https://pastebin.com/pWM1y61d)
- [tmm88](https://pastebin.com/3VHV0cY1)
- [jf99](https://pastebin.com/gLTMV3kb)
- [vkatsikaros](https://gist.github.com/vkatsikaros/551c284c609a62a4fa8035b612e0838f)
- [est31](https://gist.github.com/est31/166021fc4a9efc6f82caf9ba50498be9)

I included the raw version of the sentences (except for tmm88's sentences which included numbers before each line). Edit: I've notice too late that tmm8's contributions have a lot of phrases and incomplete sentences. Might need to be reverted?

### CODE REVIEW

**Comment by mikehenrty** [comment]
Thanks for putting these together @mds3dstn71 !

I have to read each sentence before I can add it. I'll try to take a look at this sometime today. But formatting this as a PR is super helpful. Ty

**Comment by mikehenrty** [comment]
@janetcshih can you review these?

File: server/data/ceejaydeepee.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Moons sped past the window in a receding, relativistic blur.
+The coronal mass ejection from the star masked the signal.
+Why does my bee always come last in the race?
+Starting from the bottom means that you appreciate the full spectrum of experience.
+Nearly seven thousand people had begun to realise just what a catastrophic mistake they had made.
+And in the end, everyone escaped with only minor abrasions.
+What kind of apparition is this?
+She was procrastinating so much.
+Chlorophyll absorbs most wavelengths of light apart from green.
+The garden was alive with insects; dragonflies buzzed chaotically above the pond.
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/ceejaydeepee.txt]: i'll split into 2 sentences where I find these

File: server/data/est31.txt [hunk 2]
```diff
@@ -98,3 +98,106 @@ The shop is open from 8am to 8pm.
...
+Solar seems like a good replacement.
+The solar industry has been on the rise recently.
+Prices for solar panels have dropped in past years.
+In many areas, there is enough sun so that solar is cheaper than anything else.
+Critics say that this is only possible through subsidies.
+This view is wrong though.
+Other engergy sources receive subsidies as well, often much higher.
+Also, oil and coal have hidden costs that others have to pay.
+That's often due to their nature.
+The term for these hidden costs is "Externalities".
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/est31.txt]: let's leave them for now

File: server/data/jf99.txt [hunk 3]
```diff
@@ -696,3 +696,308 @@ Sentences are built of nouns, verbs and sometimes adjectives.
 To find out the exact threshold, we need to repeat this in a reproducible manner.
 Please stop the recording by pressing the button again, when you have finished reading this sentence.
 You can verify and re-record your own recordings before submitting them.
+It takes two to tango.
+Lima is the capital of Peru.
+Oscar is dancing foxtrot together with Clara.
+In colloquial language the term Yankee is used for U.S. citizens.
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/jf99.txt]: good flag. we'll keep it like it is for now

File: server/data/jf99.txt [hunk 4]
```diff
@@ -696,3 +696,308 @@ Sentences are built of nouns, verbs and sometimes adjectives.
...
+Helena tied up her spouse, for whatever reason.
+We got off with a slap on the wrist.
+The new patch is less invasive than the old one, but still causes regressions.
+How is Mozilla going to handle ambiguities like queue and cue?
+Context is needed.
+Also, will numbers be written as digits or as words?
+Shawty, you look jaw-dropping in that jersey.
+The grim reaper is waiting for you, reckless driver!
+They found all sorts of drugs at Howard's house: Cocaine, meth, heroine...
+undefined reference to 'swizzle'
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/jf99.txt]: programming joke. short phrases are allowed, but we like mostly proper sentences

File: server/data/jf99.txt [hunk 5]
```diff
@@ -696,3 +696,308 @@ Sentences are built of nouns, verbs and sometimes adjectives.
...
+Most of the energy will be absorbed by kinetic friction.
+With that new haircut you look ridiculous.
+Completing objectives will unlock new skins and skills.
+Ransomware is the kind of malware that blackmails you after encrypting your files.
+It takes decades until you're qualified to join the granny gang.
+Last night my little sister crawled into my bed, after she'd had a nightmare.
+You can choose either yes or nope.
+You can reduce the quality of the anisotropic filter to gain some performance.
+A rooster will attack other cocks that enter its territory.
+"How many hens are there for one cock?", I asked the farmer.
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/jf99.txt]: thanks for the flags, i think we will keep these

File: server/data/tmm88.txt [hunk 6]
```diff
@@ -0,0 +1,51 @@
+as above, so below. as within so without. as the universe so the soul.
+the existence is a set of networks, modeled hierarchically within infinite dimensions.
+I look through the infinity of your face, and contemplate the latent sense in the dietary static of the pulsar of hours
+as i crawl through a glimmer of hollow sphere, i resemble the outcast of fear.
+we all share a common denominator: love, the supreme force that reasembles all the there is, there was, and will be
+I find myself, I find you !!! Find it, find it.
+Meet, meet, meet;
+There, whitewashed, between the meso-vitriolic flame Of the latent sense in the mecanosphere of love.
+An exchange of looks. An exchange of gestures.
```
[the remark this answers is not in the document]
Comment (mikehenrty) [reply server/data/tmm88.txt]: yeah, we'll just remove this whole file

**Comment by mikehenrty** [comment]
Thanks for the very thorough review @janetcshih ! I'll make the changes you suggested.

**Comment by mikehenrty** [comment]
Thank you @mds3dstn71!

**Comment by jf99** [comment]
Thanks for correcting and merging! For the following sentence, I'd strongly propose to use a dash or some other kind of sentence interruption:
`I can reassure you, that we won't retreat until the oil until the peace is secured.`
This is meant to be a Freudian slip.

File: server/data/jf99.txt [hunk 7]
```diff
@@ -696,3 +696,308 @@ Sentences are built of nouns, verbs and sometimes adjectives.
...
+My lawn-mower is broken.
+You two in the last row, stop giggling!
+When it comes to the crunch, our company will become insolvent.
+These machines are built for number crunching.
+The life of that bungee jumper dangles on a string, in the truest sense of the word.
+Did the dog sniff anyone out yet?
+They mocked him by imitating all of his actions.
+Wow, look how deep this canyon is!
+Can you see the beacon at the top of the wind turbine?
+Alice, who the fuck is Bob?
```
[the remark this answers is not in the document]
Comment (iveskins) [reply server/data/jf99.txt]: Came across this "fuck" today. 
this seems at odds with the discussion here. https://discourse.mozilla.org/t/offensive-language/20238

**Pull request merged**